src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -246,87 +234,68 @@ export class ViolationTicketService {
-    const data: DialogOptions = {
-      titleKey: err.error.title,
-      actionType: 'warn',
-      messageKey: 'error1',
-      // err.error.errors.file ? err.error.errors.file[0] : err.error.errors[0],
-      actionTextKey: 'Ok',
-      cancelHide: true,
-    };
-    return this.dialog.open(ImageTicketNotFoundDialogComponent, { data })
+  private isErrorMatch(err, msg, exactMatch = true) {
+    return exactMatch ? err.error.errors?.includes(msg) : err.error.errors?.filter(i => i.indexOf(msg) > -1).length > 0;
   }
-  errorSeenarioOne() {
+
+  private openImageTicketNotFoundDialog(title, key) {
     const data: DialogOptions = {
-      titleKey: "Your ticket image could not be read",
+      titleKey: title,
       actionType: 'warn',
-      messageKey: 'error1',
-      // err.error.errors.file ? err.error.errors.file[0] : err.error.errors[0],
+      messageKey: key,
       actionTextKey: 'Ok',
       cancelHide: true,
     };
     return this.dialog.open(ImageTicketNotFoundDialogComponent, { data })
   }
-  errorSeenarioTwo() {
-    const data: DialogOptions = {
-      titleKey: "Your ticket is over 30 days old",
-      actionType: 'warn',
-      messageKey: 'error2',
-      // err.error.errors.file ? err.error.errors.file[0] : err.error.errors[0],
-      actionTextKey: 'Ok',
-      cancelHide: true,
-    };
-    return this.dialog.open(ImageTicketNotFoundDialogComponent, { data })
+
+  private openErrorScenarioOneDialog() {
+    return this.openImageTicketNotFoundDialog("Your ticket image could not be read", "error1");
   }
-  errorSeenarioThree() {
-    const data: DialogOptions = {
-      titleKey: "Invalid ticket type",
-      actionType: 'warn',
-      messageKey: 'error3',
-      // err.error.errors.file ? err.error.errors.file[0] : err.error.errors[0],
-      actionTextKey: 'Ok',
-      cancelHide: true,
-    };
-    return this.dialog.open(ImageTicketNotFoundDialogComponent, { data })
+
+  private openErrorScenarioTwoDialog() {
+    return this.openImageTicketNotFoundDialog("Your ticket is over 30 days old", "error2");
   }
-  inValidTicketDateDialog() {
-    const data: DialogOptions = {
-      titleKey: 'more than 30 days old',
-      actionType: 'warn',
-      messageKey: "error2",
-      actionTextKey: 'Ok',
-      cancelHide: true,
-    };
-    return this.dialog.open(ImageTicketNotFoundDialogComponent, { data })
+
+  private openErrorScenarioThreeDialog() {
+    return this.openImageTicketNotFoundDialog("Invalid ticket type", "error3");
   }
-  public onTicketNotFound(): void {
-    this.notFound = true;
-    this.dialog.open(TicketNotFoundDialogComponent);
+
+  private openInValidTicketDateDialog() {
+    return this.openImageTicketNotFoundDialog("more than 30 days old", "error2");
+  }
+
+  private dateDiff(givenDate) {
+    var diffYear = (new Date().getTime() - new Date(givenDate).getTime()) / 1000;
+    diffYear /= (60 * 60 * 24);
+    return Math.abs(Math.round(diffYear));
+  }
+
+  private checkSize(fileSize) {
```

Comment:
These functions do not have any comments. I have no idea what this function is doing. I realize this code was already there. It looks like it is checking if the file sizes is greater than X or less than Y. From my tests, it seems if the file size is less than or equal to 10491002, this function return true, otherwise false. To me, this could be simplified to

 `return fileSize > 0 && fileSize <= 10485760; // less or equal to 10MB ` 

or

 `return fileSize > 0 && fileSize <= (10 * 1024 * 1024); // less or equal to 10MB `